x-pack/plugins/observability/public/pages/overview/index.tsx

```diff
@@ -52,6 +52,9 @@ export function OverviewPage({ routeParams }: Props) {
   const { data: newsFeed } = useFetcher(() => getNewsFeed({ core }), [core]);
 
   const theme = useContext(ThemeContext);
+
+  // read time from state and update the url
+  const sharedTimeState = plugins.data.query.timefilter.timefilter.getTime();
```

Comment:
I see you read the time from `timefilter`. Don't you also need to call `setTime` somewhere?

Reply:
I read it from here, then pass it down to DatePicker component. @shahzad31 is working on a PR that refactors that part moving this to the component, so maybe we could wait until he's done with his changes to apply mine https://github.com/elastic/kibana/pull/78976. WDYT?

Reply:
i think don't wait for me, please go ahead and merge yours, i will resolve mine conflicts if any arise from yours.

Reply:
Okay, so you'll update this PR when #78976 is merged?

Reply:
yeah, that's the plan.

Reply:
Btw. I wrote my comment before seeing @shahzad31's (Github is soooo bad at loading dynamic content...).
So sounds like you can just go ahead and implement the `setTime` functionality in this PR.

Reply:
it's upto @cauemarcondes , how he wants to handle this :)

Reply:
> So sounds like you can just go ahead and implement the setTime functionality in this PR

Done!